### PR TITLE
Add curl examples to notebooks

### PR DESCRIPTION
**Pull request by yubozhao**
What changes were proposed in this pull request?
------------------------------------------------
Add curl commands to Rest API service example
Does this close any currently open issues?
------------------------------------------
https://github.com/bentoml/BentoML/issues/76

How was this patch tested?
--------------------------

### CODE REVIEW

File: examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb [hunk 1]
```diff
@@ -277,12 +277,25 @@
...
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "!curl -i \\\n",
+    "--header \"Content-Type: application/json\" \\\n",
+    "--method POST \\\n",
```
Comment (parano) [inline examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb]: shouldn't this be `--request POST`?

File: examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb [hunk 2]
```diff
@@ -277,12 +277,25 @@
...
+    "--data '{\"text\": \"some new text, sweet noodles\"}'\n",
```
Comment (parano) [inline examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb]: missed a `\` here

Comment (parano) [inline examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb]: I don't think this is the expected input format. could you test it out and fix this?

File: examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb (hunk 1 above)
Comment (yubozhao) [reply examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb]: Yep, I will recheck this PR and submit again

**Pull request merged**